Automattic/newspack-plugin
# feat(ia-listings): IA for Listings Plugin

### All Submissions:






### Changes proposed in this Pull Request:



This code completes Asana Task: "Newspack Information Architecture > IA Listings".  

Screenshot of Menu:

![ia-listings-changes](https://github.com/user-attachments/assets/9d5f73ca-dc85-48c3-b3aa-36ce9727bb29)

* Menu position and icon updated.
* All sub menu items have the new Admin Header.
* Links to categories/tags removed.
* Link for Marketplace moved above Generic ( SEE RELATED: https://github.com/Automattic/newspack-listings/pull/487 )

Screenshot of Dashboard:

![ia-listings-dashboard](https://github.com/user-attachments/assets/aa94b045-497b-4857-afcc-1300abed525d)

* "Cards" are clickable to Listings menu items.
* When Listings Plugin is inactive, entire section and cards are removed.

### How to test the changes in this Pull Request:

1. Pull branch.
2. `npm run build`
3. Activate Newspack plugin.
4. Install/activate Newspack Listings Plugin.
5. Verify screenshots and notes above.
6. Deactivate Listings plugin, verify dashboard no longer shows Listings section.

### Note about Related PR:

The code to move the submenu link for Marketplace above Generic is done in the Listings Plugin. ( SEE RELATED: https://github.com/Automattic/newspack-listings/pull/487 ) 

If we are unable to approve that PR, then we'll need to approve this PR keeping Generic above Marketplace (this is how Listings Plugin does it already), or we'll need to update this PR with a solution.




### Other information:
